src/error/GraphQLError.ts

```diff
@@ -275,7 +275,7 @@ export interface GraphQLFormattedError {
    * Reserved for implementors to extend the protocol however they see fit,
    * and hence there are no additional restrictions on its contents.
    */
-  readonly extensions?: { [key: string]: unknown };
+  readonly extensions?: GraphQLErrorExtensions;
```

Comment:
What do you think about...

```suggestion
  readonly extensions?: GraphQLFormattedErrorExtensions;
```

with the new interface:

```ts
/**
 * Custom extensions
 *
 * @remarks
 * Use a unique identifier name for your extension, for example the name of
 * your library or project. Do not use a shortened identifier as this increases
 * the risk of conflicts. We recommend you add at most one extension field,
 * an object which can contain all the values you need.
 */
export interface GraphQLFormattedErrorExtensions {
  [attributeName: string]: unknown;
}
```

Reply:
I'm not convinced of the point of having two different types, but it won't cost any more.

I'll do it